RecoLocalCalo/HGCalRecProducers/python/hgcalLayerClusters_cff.py

```diff
@@ -6,18 +6,21 @@
 
 from RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi import HGCalUncalibRecHit
 
-from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import fC_per_ele, hgceeDigitizer, hgchebackDigitizer, hfnoseDigitizer
+from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import fC_per_ele, HGCAL_noises, hgceeDigitizer, hgchebackDigitizer, hfnoseDigitizer
 
 hgcalLayerClusters = hgcalLayerClusters_.clone()
 
 hgcalLayerClusters.timeOffset = hgceeDigitizer.tofDelay
 hgcalLayerClusters.plugin.dEdXweights = cms.vdouble(dEdX.weights)
+#With the introduction of 7 regional factors (6 for silicon plus 1 for scintillator),
+#we extend fcPerMip (along with noises below) so that it is guaranteed that they have 6 entries.
 hgcalLayerClusters.plugin.fcPerMip = cms.vdouble(HGCalUncalibRecHit.HGCEEConfig.fCPerMIP + HGCalUncalibRecHit.HGCHEFConfig.fCPerMIP)
 hgcalLayerClusters.plugin.thicknessCorrection = cms.vdouble(HGCalRecHit.thicknessCorrection)
 hgcalLayerClusters.plugin.sciThicknessCorrection = HGCalRecHit.sciThicknessCorrection
 hgcalLayerClusters.plugin.deltasi_index_regemfac = HGCalRecHit.deltasi_index_regemfac
 hgcalLayerClusters.plugin.fcPerEle = cms.double(fC_per_ele)
-hgcalLayerClusters.plugin.noises = cms.PSet(refToPSet_ = cms.string('HGCAL_noises'))
+#Extending noises as fcPerMip, see comment above.
+hgcalLayerClusters.plugin.noises = cms.vdouble(HGCAL_noises.values + HGCAL_noises.values)
```

Comment:
```suggestion
hgcalLayerClusters.plugin.noises = HGCAL_noises.values + HGCAL_noises.values
```
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

it looks like this file did not make it to #30147 and the "drop type specs" migration task:
- in the lines above the `hgcalLayerClusters = hgcalLayerClusters_.clone()` appears separately from the parameters modified later
- in the lines below inside a clone `nHitsTime = cms.uint32(3)` from the old code still appears with type details

@jeongeun please check/clarify what happened.

I made a few comments to this file mainly for the updated lines, but it looks like our migration missed this file somehow